contents.json

```diff
@@ -17059,6 +17059,20 @@
       ],
       "date_added": "Sep 3 2018",
       "suggested_by": "@archagon"
+    },
+    {
+      "title": "Ragnarok",
+      "category-ids": ["game"],
+      "description": "A simple game application for the practice of beginners in Swift-programming",
+      "source": "https://github.com/riley-usagi/SwiftAdvance",
+      "itunes": "",
```

Comment:
@riley-usagi Could you remove this `"itunes": "",`? You don't need this line if an app is not in the Apple Store.